### ISSUE TITLE
[Issue] crash with Valkyrien Skies

### ISSUE THREAD

**Issue by gatoborrachon** [necauqua/chiseled-me#107]
When I try to use this mod with mod *[Valkyrien Skies](https://www.curseforge.com/minecraft/mc-mods/valkyrien-skies) (and its 2 modules: [VS-World](https://www.curseforge.com/minecraft/mc-mods/valkyrien-skies-world) and [VS-Control](https://www.curseforge.com/minecraft/mc-mods/valkyrien-skies-control))* and i try to load a new/existing world, the game crashes. 

this is the same behavior that [this issue](https://github.com/necauqua/chiseled-me/issues/60) said in 2020

**Additional information:**
- Crash Report: https://pastebin.com/qcgy2qad
- Log: [latest.log](https://github.com/necauqua/chiseled-me/files/6666577/latest.log)
- `minecraft_profile_folder/logs/debug.log` file: [debug.log](https://github.com/necauqua/chiseled-me/files/6666579/debug.log)

**Comment by gatoborrachon** [comment]
I also want to add that when i tried Chiseled Me in a custom modpack, i got this crash report instead: (more related to Thaumcraft): https://pastebin.com/NWw1FSf1
(Same crash with only Chiseled Me, Valkyrien Skies and Thaumcraft: https://pastebin.com/hfdHvze7)

if you manage to fix this issue (only between Chiseled Me and Valkryen Skies) i will try to load Thaumcraft and see if that issue is fixed too (I suppose it should, there are not issues between Valkyrien Skies and Thaumcraft, nor issues between Chiseled Me and Thaumcraft)

**Comment by necauqua** [comment]
Yeah I see that CM and VS are actually incompatible - they try to modify the same thing and that's an incompatibility.
I'll look into it once I'm back on working on cm (around mid summer), but the priority is 1.16 so I can't promise this will ever get fixed.

**Comment by gatoborrachon** [comment]
oh, sad to hear that, but no problem, i hope it is "easy" to fix, many thanks (i know that it never is)

take the necessary time, no problem

**Comment by necauqua** [comment]
This one is one of those that should be actually easy to fix, yes. But ofc should be and will be are different things :)
